src/Serilog.Extensions.Logging/Serilog.Extensions.Logging.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Serilog provider for Microsoft.Extensions.Logging</Description>
     <VersionPrefix>1.4.1</VersionPrefix>
```

Comment:
I think we could rev this to `2.0.0` to signal the update supporting the new MEL.